src/server/routes/backend-wallet/sign-transaction.ts

```diff
@@ -21,7 +25,7 @@ const requestBodySchema = Type.Object({
     gasPrice: Type.Optional(Type.String()),
     data: Type.Optional(Type.String()),
     value: Type.Optional(Type.String()),
-    chainId: Type.Optional(Type.Integer()),
```

Comment:
Is this gonna break existing users calling without a chain id?

Reply:
yes, open to any way to support pre-eip155 signatures as well. I couldn't find a way with the current SDK.

Reply:
Tbh signing a tx without a chain id is kinda sus. And I guess it will only break ppl when they upgrade?

Maybe we make it really clear in the release notes